src/site/xdoc/checks/coding/avoidinlineconditionals.xml.template

```diff
@@ -14,7 +14,7 @@
         </div>
         <source>
 String a = getParameter("a");
-String b = (a==null || a.length()&lt;1) ? null : a.substring(1);
+String b = (a == null || a.length() &lt; 1) ? null : a.substring(1);
```

Comment:
fix spacing: its important for the docs only to violate the actual item and not others too. This is about `inline` and not `whitespace`.

<img width="456" alt="image" src="https://github.com/user-attachments/assets/35a6c3dc-46b0-4174-bd45-cc421216d982" />